plans/main.fmf

```diff
@@ -0,0 +1,64 @@
+summary: Run Python Podman Tests
+
+discover:
+    how: fmf
+execute:
+    how: tmt
+prepare:
+  - name: pkg dependencies
+    how: install
+    package:
+      - make
+      - podman
+      - python3-pip
+      - python3.9
+      - python3.10
+      - python3.11
+      - python3.12
+      - python3.13
+
+  - name: pip dependencies
+    how: shell
+    script:
+      - pip3 install -r test-requirements.txt
+
+  - name: ssh configuration
+    how: shell
+    script:
+        - ssh-keygen -t ecdsa -b 521 -f /root/.ssh/id_ecdsa -P ""
+        - cp /root/.ssh/authorized_keys /root/.ssh/authorized_keys%
+        - cat /root/.ssh/id_ecdsa.pub >>/root/.ssh/authorized_keys
+
+/upstream:
+    /lint:
+        summary: Run Python Podman Tests on upstream PRs
+        discover+:
+            test: /tests/lint
+
+    /integration_tests:
+        summary: Run Python Podman Tests on upstream PRs
+        discover+:
+            test: /tests/all_integration
+
+    /unittest:
+        summary: Run Python Podman Unit Tests on upstream PRs
+        discover+:
+            test: /tests/unittest
+
+    /integration:
+        summary: Run Python Podman Integration Tests on upstream PRs
+        discover+:
+            test: /tests/integration
+
+    adjust+:
+        enabled: false
+        when: initiator is not defined or initiator != packit
+
+/downstream:
+    summary: Run Python Podman Tests on bodhi / errata and dist-git PRs
+    discover+:
+        test: /tests/integration_tests
```

Comment:
When these actually run downstream, I guess they'll need access to the podman-py test files in the upstream source, and I don't think we're packaging those yet.

You could either do the fetch tests from upstream URL that you've done on dnf IIRC. Or else you could add: 
```
  dist-git-source: true
  dist-git-merge: true
```
in this discover+ section.

Might be worth creating a draft PR on python-podman dist-git and verifying before we merge this.

Reply:
Actually, scratch my previous comment. Let's get ourselves comfortable upstream. We can deal with downstream later.